src/WorkflowCore/Services/BackgroundTasks/EventConsumer.cs

```diff
@@ -13,7 +13,7 @@ internal class EventConsumer : QueueConsumer, IBackgroundTask
         private readonly IPersistenceProvider _persistenceStore;
         private readonly IDistributedLockProvider _lockProvider;
         private readonly IDateTimeProvider _datetimeProvider;
-        protected override int MaxConcurrentItems => 2;
+        protected override int MaxConcurrentItems => WorkflowOptions.MinimumNumberOfConcurrentItems;
```

Comment:
This doesn't make sense to me... is this a typo?

Reply:
That was intentional, because MaxConcurrentItems  as a function of the number 2 was showing in two places, the EventConsumer and QueueConsumer.  And then I added a 3rd place it was showing up, with the WorkflowOptions.  I was concerned the number 2 was becoming a magic number, but this change doesn't add a ton of value, so I can revert it if its problematic.